scroll-animations-1/Overview.bs

```diff
@@ -7,13 +7,14 @@ Work Status: exploring
 Group: CSSWG
 URL: https://drafts.csswg.org/scroll-animations-1/
 ED: https://drafts.csswg.org/scroll-animations-1/
-Shortname: scroll-animations
+Shortname: scroll-animations-1
 Abstract: Defines an API and markup for creating animations that are either
           triggered by or tied to the scroll offset of a scroll container.
 Editor: Brian Birtles <bbirtles@mozilla.com>
 Editor: Botond Ballo <botond@mozilla.com>
 Editor: Stephen McGruer <smcgruer@google.com>
 Editor: Antoine Quint <graouts@apple.com>
+Editor: Jordan Taylor <jortaylo@microsoft.com>
```

Comment:
FYI you don't need to be an editor to make this change.

https://github.com/w3c/csswg-drafts/blob/master/CONTRIBUTING.md#after-discussion